lib/ActivityStream.jsm

```diff
@@ -192,6 +192,10 @@ const PREFS_CONFIG = new Map([
     title: "A comma-delimited list of search shortcuts that have previously been pinned",
     value: "",
   }],
+  ["improvesearch.handoffToAwesomebar", {
+    title: "Should the search box handoff to the Awesomebar?",
+    value: true,
```

Comment:
Should we land this enabled on Nightly?

Reply:
I brought it up in my 1:1, Tim thinks we should ship it enable on Nightly and Beta. Then we'll do a study in Release before rolling out. If something goes wrong in Nightly it's easy enough to disable it with a one liner patch.

Reply:
Ok, sounds good to me 👍

Reply:
Let's go with on for now and talk about our plan for testing it (I think we should be careful about the possible impact to our search numbers this could have)

If you're going to land this with just `true`, would you mind filing another issue to turn it off for release etc. so we dont forget about that work?